src/js/editor/event-manager.js

```diff
@@ -210,6 +217,43 @@ export default class EventManager {
     this._updateModifiersFromKey(key, {isDown:false});
   }
 
+  compositionstart() {
```

Comment:
why not use camelCase instead?
```suggestion
  compositionStart() {
```

Reply:
oops, that's the event name.